File: evento/src/main/java/com/myevent/eventos/entity/Horario.java
package com.myevent.eventos.entity;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import jakarta.persistence.JoinColumn;
import jakarta.persistence.ManyToOne;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@Entity
public class Horario {

	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	private Long id;
	
	private String data;
	private String horarioInicio;
	private String horarioFim;
	
    @ManyToOne
    @JoinColumn(name = "local_id")
    private Local local;
}

File: evento-app/evento-app/src/app/sistema/evento/Horario.ts
export class Horario{
    id!: number
    data: string
    horarioInicio: string
    horarioFim: string
    localId: number

    constructor( 
        data: string,
        horarioInicio: string,
        horarioFim: string,
        localId: number){
            this.data = data;
            this.horarioInicio = horarioInicio;
            this.horarioFim = horarioFim;
            this.localId = localId;
    }
}